src/App.tsx

```diff
@@ -1,61 +1,117 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { getUserId } from './services/User';
+
+const initialTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserId(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>(initialTodos);
+  const [title, setTitle] = useState('');
+  const [hasTittleError, setHasTittleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [hasSelectError, setHasSelectError] = useState(false);
+
+  const addTodo = (newTodo: Omit<Todo, 'id'>) => {
+    const todoToAdd: Todo = {
+      ...newTodo,
+      id: Math.max(...todos.map(todo => todo.id)) + 1,
+    };
+
+    setTodos(currentTodos => [...currentTodos, todoToAdd]);
+  };
+
+  const handleChangeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setHasTittleError(false);
```

Comment:
The function `setHasTittleError` is misspelled. It should be `setHasTitleError` to match the corrected state variable name.